web/src/test/resources/com/graphhopper/http/resources/cargo_bike.yml

```diff
@@ -0,0 +1,26 @@
+base: bike
+
+vehicle_height: 2.3
+vehicle_width: 1.2
+
+# let's assume cargo bikes are e-bikes and we should lower speed to prefer shorter and not faster routes automatically
+max_speed:
+  road_class:
+    primary: 28
+
+# if no other condition matches
+max_speed_fallback: 25
+
+priority:
+  # exclude steps
+  road_class:
+    steps: 0
+
+  surface:
+    sand: 0.5
+  # prefer better tracks
+  track_type: { other: 1.1, grade1: 1.1 }
+  # prefer official bike routes
+  bike_network: { other: 0.5 }
```

Comment:
Here we rather want to *prefer* bike_network, right? So it should be rather 1.5 than 0.5?

Reply:
Ha I get it you did this on purpose, now I am excited to try if I can find some routes that change depending on this number :)

Reply:
No, but surely this needs more explanation and is similar to the toll stuff. We have multiple values like `international` or `national`, see RouteNetwork enum. And we want to prefer them all or just reduce priority of `other`, which is the default.

Reply:
Sorry, my comment is not valid - did not see your last comment :) ... it seems github is still not able to properly refresh this via ajax.

Reply:
Ah it says bike_network: other: 0.5 so everything **except** the official bike networks gets a lower priority. Sorry my fault I kind of skipped the 'other' when reading this. Is there some documentation what kind of values the different encoded values have?

Reply:
Kind of as Java code :) 

https://github.com/graphhopper/graphhopper/blob/master/core/src/main/java/com/graphhopper/routing/profiles/RouteNetwork.java#L22

Reply:
Ok that does not count :)

Reply:
We could even use a json/swagger spec? (I did not find something similar for yaml :/ )

Or even better an editor with autocompletion so users get the information very fast :)

Reply:
Have implemented this via the /info endpoint and the "auto completion"